query_object.rb

```diff
@@ -1,45 +1,58 @@
-module YourSuperModule
+module ParamsMethods
   def self.included(host_class)
```

Comment:
Usually we use `base` here not `host_class`

Reply:
Fixed